extensions/blocks/map/map.php

```diff
@@ -25,7 +25,85 @@
 function jetpack_map_block_load_assets( $attr, $content ) {
 	$api_key = Jetpack_Options::get_option( 'mapbox_api_key' );
 
+	if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+		static $map_block_counter = [];
+
+		$id = get_the_ID();
+		if ( ! $map_block_counter[ $id ] ) {
+			$map_block_counter[ $id ] = 0;
+		}
+		$map_block_counter[ $id ]++;
+
+		$iframe_url = add_query_arg(
+			array(
+				'map-block-counter' => $map_block_counter[ get_the_ID() ],
+				'map-block-post-id' => get_the_ID(),
+			),
+			get_permalink()
+		);
+
+		$placeholder = preg_replace( '/(?<=<div\s)/', 'placeholder ', $content );
+
+		return sprintf(
+			'<amp-iframe src="%s" width="%d" height="%d" layout="responsive" allowfullscreen sandbox="allow-scripts">%s</amp-iframe>',
+			esc_url( $iframe_url ),
+			absint( Jetpack::get_content_width() ),
+			absint( Jetpack::get_content_width() * 0.75 ),
```

Comment:
There's actually no need to use `content_width` here. Responsive only cares about the aspect ratio. So you could just make the above:

`width="100" height="75"`

Reply:
Knocked it all the way down to 4:3 in https://github.com/Automattic/jetpack/pull/13405/commits/fe51cb4ac0e62d57d62898f2317fc66d62f1d78e